packages/webpack-cms-plugins/HubSpotAutoUploadPlugin.js

```diff
@@ -35,34 +35,29 @@ class HubSpotAutoUploadPlugin {
     setLogger(webpackLogger);
     compiler.hooks.afterEmit.tapPromise(pluginName, async compilation => {
       Object.keys(compilation.assets).forEach(filename => {
-        const asset = compilation.assets[filename];
-        // assets with `emitted = false` haven't changed since the previous compilation
-        if (asset.emitted) {
```

Comment:
In my testing, the only assets that were included in a compilation were assets that get written. I'm curious if I'm missing something else here, otherwise it seems safe to remove this.

cc @gcorne since you worked on v1 of this

Reply:
I think that this was intended to catch situations where the file had the same contents as the previous compilation to avoid unnecessary uploads.

Reply:
I'll give this another test

Reply:
Did some more testing on this. It can't seem to reproduce a case where there is an `asset` in `compilation.assets` with `emitted: false`. Only files that were written are included in `assets`.

It's worth noting that in my test project (https://github.com/HubSpot/cms-event-registration) these files bundled for the two main entry points were updated after any change, even if it was unrelated. So those did upload even if the file content wasn't different.